source/reference/array.rst

```diff
@@ -236,7 +236,200 @@ Unevaluated Items
 
 |draft2019-09|
 
-Documentation Coming Soon
+The ``unevaluatedItems`` keyword selects any data types not evaluated
+by an ``items``, ``prefixItems``, or `contains` keyword. Just as
+unevaluated"properties" affect only "properties" in an object, only
```

Comment:
Bit more common I think to use emphasis -- e.g. unevaluated*Properties* -- rather than quotes.